packages/web/src/components/gcds-link/stories/gcds-link.stories.js

```diff
@@ -0,0 +1,211 @@
+    },
+  },
+
+  argTypes: {
+    // Props
+    size: {
+      control: { type: 'radio' },
+      options: ['regular', 'small', 'inherit'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'regular' },
+      },
+    },
+    display: {
+      control: 'select',
+      options: ['inline', 'inline-block', 'block'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'inline' },
+      },
+    },
+    href: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    rel: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    target: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '_self' },
+      },
+    },
+    external: {
+      control: 'boolean',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    download: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    type: {
+      control: { type: 'text' },
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+
+    // Slots
+    default: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Slots | Fentes',
+      },
+    },
+  },
+};
+
+const Template = args =>
+  `
+This is an example of
+<!-- Web component code (Angular, Vue) -->
+<gcds-link ${args.display ? `display="${args.display}"` : null} ${
+    args.href ? `href="${args.href}"` : null
+  } ${args.rel ? `rel="${args.rel}"` : null} ${
+    args.target ? `target="${args.target}"` : null
+  } ${args.size != 'regular' && args.size ? `size="${args.size}"` : null} ${
+    args.external ? `external` : null
+  } ${args.download ? `download="${args.download}"` : null} ${
+    args.type ? `type="${args.type}"` : null
+  }>${args.default}</gcds-link>
+
+<!-- React code -->
+<GcdsLink ${args.display ? `display="${args.display}"` : null} ${
+    args.href ? `href="${args.href}"` : null
+  } ${args.rel ? `rel="${args.rel}"` : null} ${
+    args.target ? `target="${args.target}"` : null
+  } ${args.size != 'regular' && args.size ? `size="${args.size}"` : null} ${
+    args.external ? `external` : null
+  } ${args.download ? `download="${args.download}"` : null} ${
+    args.type ? `type="${args.type}"` : null
+  }>${args.default}</GcdsLink>
+link.
+`.replace(/ null/g, '');
+
+const TemplateSizes = () => `
+<!-- Web component code (Angular, Vue) -->
+<gcds-link href="" size="regular">This is a regular link.</gcds-link>
+<gcds-link href="" size="small">This is a small link.</gcds-link>
+
+<!-- React code -->
+<GcdsLink href="" size="regular">This is a regular link</GcdsLink>
```

Comment:
Can we remove `size="regular"` since `regular` is the default value for the `size prop, users don't have to add this.